configs/config.json

```diff
@@ -1,11 +1,11 @@
 {
   "map": {
-    "projection": "EPSG:3857",
+    "projection": "EPSG:900913",
```

Comment:
@dsuren1 where this change come from? Correct me if I'm wrong, but it doesn't seem related to any of the issues listed in the PR description and it is not present even in master branch.

Reply:
I observed this [issue](https://github.com/georchestra/mapstore2-georchestra/issues/681) still occurs when EPSG:900913 & EPSG:3857 are interchangeably used. However, georchestra’s other supported projection lists are not affected, hence modifying the default config projection is enough to fix. 
**Note**: We need this on georchestra master too. I will create one.

Reply:
im not a projection expert and im pretty sure you guys well know that, but from my understanding the usage of the unofficial EPSG:900913 was discouraged in favor of the official EPSG:3857 since 2009..

cf:
- https://epsg.io/900913
- https://wiki.openstreetmap.org/wiki/Web_Mercator
- https://en.wikipedia.org/wiki/Web_Mercator_projection#Identifiers

can we avoid this ?

Reply:
Sure. This seems to be caused when transforming a polygon into a circle when the circle is geodesic. Since few projections seem to work fine because of the minuscule difference in radius, it was overlooked. Will address it.

Reply:
@dsuren1 we have to review the fix on the MS side for this, of course.

Reply:
PR with fix added on [MS](https://github.com/geosolutions-it/MapStore2/pull/9938). The config change reverted. So the rest can be reviewed. Thanks